themes.gitconfig

```diff
@@ -175,6 +175,46 @@
     blame-format = "{author:<18} ({commit:>7}) ┊{timestamp:^16}┊ "
     blame-palette = "#101010 #200020 #002800 #000028 #202000 #280000 #002020 #002800 #202020"
 
+[delta "pablospe"]
```

Comment:
Hi @pablospe, thanks for this! Could you follow this instruction at the top of the file:

```
# 1. The name of the theme must be the name of some sort of wild organism:
#    mammal, bird, plant, mollusk -- whatever. It can be in any language.
#
```

Reply:
`Colibri`? (Spanish for hummingbird)

Reply:
It might be better to sort this in ascending order. I will make another commit for this.

Reply:
ok, I realized afterwards that they were not arranged alphabetically.